catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliConstants.java

```diff
@@ -22,9 +22,11 @@
 
 public class NsiliConstants {
 
+  private NsiliConstants() {}
```

Comment:
What is the point of this?

Reply:
To prevent construction of a singleton class with only static methods.

Reply:
Yeah it was a small finding that I figured I would just add while I was in here.